src/modules/render.js

```diff
@@ -0,0 +1,51 @@
+const ALL_MEALS = document.querySelector('#all-meals');
+
+const RENDER = (meals) => {
+  meals.forEach((meal) => {
+    const MEAL = document.createElement('div');
+    MEAL.className = 'meal';
+    ALL_MEALS.appendChild(MEAL);
+
+    const MEAL_IMAGE = document.createElement('img');
+    MEAL_IMAGE.className = 'meal-image';
+    MEAL_IMAGE.src = meal.image;
+    MEAL_IMAGE.alt = meal.name;
+    MEAL.appendChild(MEAL_IMAGE);
+
+    const CONTAINER_NAME_LIKES = document.createElement('div');
+    CONTAINER_NAME_LIKES.className = 'container-name-likes';
+    MEAL.appendChild(CONTAINER_NAME_LIKES);
+
+    const MEAL_NAME = document.createElement('h2');
+    MEAL_NAME.className = 'meal-name';
+    MEAL_NAME.innerHTML = meal.name;
+    CONTAINER_NAME_LIKES.appendChild(MEAL_NAME);
+
+    const CONTAINER_LIKES = document.createElement('div');
+    CONTAINER_LIKES.className = 'container-likes';
+    CONTAINER_NAME_LIKES.appendChild(CONTAINER_LIKES);
+
+    const LIKE_IMG = document.createElement('img');
+    LIKE_IMG.className = 'like-icon';
+    LIKE_IMG.src = './assets/heart-line.png';
+    LIKE_IMG.alt = 'like icon';
+    CONTAINER_LIKES.appendChild(LIKE_IMG);
```

Comment:
- Good job 👌 , but I see that the image of the like was not loaded, so you can check this problem 😊, but don't worry you are almost there 💯

Reply:
I kindly ask you to pull the last commits to see that the webpack configuration errors have been fixed so that the images can be displayed😊
![meal](https://user-images.githubusercontent.com/98427585/203831960-33e6fc32-6e05-454d-acea-d1c540683f0b.png)